content/academy/data_collection_basics/browser_devtools.md

```diff
@@ -52,4 +52,4 @@ temp1.textContent = 'Hello!';
 
 ## [](#next) Next up
 
-In this chapter we learned the absolute basics of interaction with a page using the DevTools. In the [next chapter]({{@link data_collection_basics/using_devtools.md}}), you will learn how to extract data from it. We will collect the most popular websites on the internet from <a href="https://www.alexa.com/topsites" target="_blank">Amazon Alexa</a>.
+In this chapter we learned the absolute basics of interaction with a page using the DevTools. In the [next chapter]({{@link data_collection_basics/using_devtools.md}}), you will learn how to extract data from it. We will collect data about the on-sale products on <a href="https://commerce-qd83plqbj-mstephen19.vercel.app/" target="_blank">this e-commerce website</a>.
```

Comment:
I would mention that this is not a real website, just a demo for teaching scraping, but that it uses the latest technology that modern ecommerce websites use.